WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressFormViewModel.swift

```diff
@@ -0,0 +1,74 @@
+import Yosemite
+
+final class EditAddressFormViewModel: ObservableObject {
+
+    init(address: Address?) {
+        self.originalAddress = address
+        updateFieldsWithOriginalAddress()
+    }
+
+    /// Original `Address` model.
+    ///
+    private let originalAddress: Address?
```

Comment:
I wonder if this should be a non-null address with empty content? 🤔

Reply:
Agree! Done in 6fa05527c04e3ccd49308b058a93279daf66930a + 56d9b90f8b3e0fc6409a17cfdaba9d2a96224b65.